index.test.js

```diff
@@ -30,339 +30,467 @@ describe('Equivalency', () => {
+            const { isEquivalent } = equivalency.equivalent(
+              candidate,
+              theCorrectAnswer
+            );
+
+            expect(isEquivalent).toBe(true);
+          });
+        });
+
+        it('should require a space after to in front of verbs', () => {
+          const theCorrectAnswer = 'write';
+
+          const candidate = 'towrite';
 
-        candidates.forEach(candidate => {
           const { isEquivalent } = equivalency.equivalent(
             candidate,
             theCorrectAnswer
           );
 
-          expect(isEquivalent).toBe(true);
+          expect(isEquivalent).toBe(false);
         });
       });
+    });
+  });
 
-      it('should require a space after to in front of verbs', () => {
-        const theCorrectAnswer = 'write';
+  describe('editDistance (diacritics agnostic)', () => {
+    describe('es', () => {
+      const agnosticEsEquivalency = new Equivalency()
+        .doesntMatter(Equivalency.UNICODE_NORMALIZATION)
+        .doesntMatter(Equivalency.WHITESPACE_DIFFERENCES)
+        .doesntMatter(Equivalency.CAPITALIZATION)
+        .doesntMatter(Equivalency.es.COMMON_PUNCTUATION)
+        .doesntMatter(Equivalency.es.COMMON_SYMBOLS)
+        .doesntMatter(Equivalency.COMMON_DIACRITICS)
+        .matters('-');
 
-        const candidate = 'towrite';
+      it('should return correct editDistance when strings match', () => {
+        const inputs = [
+          ['estoy bien y tu', 'estoy bien, ¿y tú?'],
+          ['como se dice', '¿Cómo se dice?'],
+          ['Tengo tres coches.', 'Tengo tres coches.'],
+          ['el combinado', 'el combínádo'],
+        ];
+        const options = { calculateEditDistance: true };
+        inputs.forEach(([s1, s2]) => {
+          const { editDistance } = agnosticEsEquivalency.equivalent(
+            s1,
+            s2,
+            options
+          );
+          expect(editDistance).toEqual(0);
+        });
+      });
 
-        const { isEquivalent } = equivalency.equivalent(
-          candidate,
-          theCorrectAnswer
-        );
+      it('should return correct editDistance when strings dont match', () => {
+        const inputs = [
+          ['e lcombinado', 'el combinado'],
+          ['e  l combinado', 'el combinado'],
+          ['e l c ombinado', 'el combinado'],
+          ['el cobminado', 'el combinado'],
+          ['el comdinabo', 'el combinado'],
+          ['apple', 'apples'],
+        ];
+        const editDistances = [1, 1, 2, 1, 2, 1];
+        const options = { calculateEditDistance: true };
+        inputs.forEach(([s1, s2], index) => {
+          const { editDistance } = agnosticEsEquivalency.equivalent(
+            s1,
+            s2,
+            options
+          );
+          expect(editDistance).toEqual(editDistances[index]);
+        });
+      });
+    });
 
-        expect(isEquivalent).toBe(false);
+    describe('en', () => {
+      const enEquivalency = new Equivalency()
```

Comment:
Probably could remove this instance and rely upon the other `enEquivalency` in this file. Thoughts? I tried to use the Spanish one for both tests, but it was causing a test to fail since I was adding the additional rule about diacritics.